src/index.html

```diff
@@ -4,10 +4,303 @@
+            <p class="person__about">Nice work, Certificates National Association of Realtors (USA)</p>
+          </div>
+          <div class="person section-four__person">
+            <div class="person__photo">
+              <img
+                src="./images/people/Ellipse 2.jpg"
+                alt="Scarlett Hawkins"
+                class="person__img"
+              >
+            </div>
+            <h2 class="person__name">Scarlett Hawkins</h2>
+            <p class="person__position">Senior Vice President</p>
+            <p class="person__about">My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart</p>
+          </div>
+          <div class="person section-four__person">
+            <div class="person__photo">
+              <img
+                src="./images/people/Ellipse 3.jpg"
+                alt="Max Henry"
+                class="person__img"
+              >
+            </div>
+            <h2 class="person__name">Max Henry</h2>
+            <p class="person__position">Chief Design Officer</p>
+            <p class="person__about">When my lovely valley of steam rises and half-day sun is above an impermeable</p>
+          </div>
+        </div>
+      </section>
+      <section id="section-5" class="main__section section-five">
+        <div class="section-five__heading">
+          <h2 class="section-five__contact-us">Contact us</h2>
+          <div class="crown section-five__crown"></div>
+        </div>
+        <div class="container section-five__container">
+          <div class="section-five__info-wrapper">
+            <div class="section-five__info">
+              <p class="section-five__way">Phone</p>
+              <p class="section-five__connection">+1 234 5555-55-55</p>
+            </div>
+            <div class="section-five__info">
+              <p class="section-five__way">Email</p>
+              <p class="section-five__connection">hello@miami.com</p>
+            </div>
+            <div class="section-five__info">
+              <p class="section-five__way">Address</p>
+              <p class="section-five__connection">400 first ave. n.<br>suite 700<br>Minneapolis, MN 55401</p>
+            </div>
+          </div>
+          <form
+            action="#"
+            class="section-five__form form"
+            method="GET"
+          >
+            <input
+              type="text"
+              name="name"
+              class="section-five__input input"
+              placeholder="Name"
+            >
+            <input
+              type="tel"
+              name="phone"
+              class="section-five__input input"
+              placeholder="Phone"
+            >
+            <textarea
+              class="section-five__input textarea"
+              name="message"
+              cols="50"
+              rows="10"
+              placeholder="Message">
+            </textarea>
```

Comment:
```suggestion
            <textarea
              class="section-five__input textarea"
              name="message"
              cols="50"
              rows="10"
              placeholder="Message"
           ></textarea>
```
ось так має з'явитись placeholder